kolena/workflow/test_sample.py

```diff
@@ -31,6 +31,13 @@ class ImageWithRegion(Image):
     region=BoundingBox(top_left=(0, 0), bottom_right=(100, 100)),
 )
 ```
+!!! note
```

Comment:
This doesn't render quite right currently:

![image](https://github.com/kolenaIO/kolena/assets/9648886/ab55fc8a-0da1-4d01-945f-db2143783ea5)

Suggested updates:

```
!!! note "Versioning for `locator` files"

    Kolena supports versioning for files stored in Amazon S3 or Google Cloud Storage. Simply enable versioning on your
    [S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Versioning.html) or [GCS](https://cloud.google.com/storage/docs/object-versioning)
    bucket and pass the `versionId` or `generation` as a part of the `locator`:

    - S3 (using `versionId`): `s3://my-bucket/example-image.png?versionId=Bv38GKqEKxwr_HYTEXYEx6TQG_4.LkAX`
    - GCS (using `generation`): `gs://my-bucket/example-image.png?generation=1701352005168905`
```

Which yields:

<img width="942" alt="Screenshot 2023-12-06 at 8 22 39 AM" src="https://github.com/kolenaIO/kolena/assets/9648886/6f090076-899b-4e79-8dbc-5e1e7ec13826">

Note that this can be previewed by running the docs server locally by running the following and visiting localhost:8999:

```
poetry run mkdocs serve -a localhost:8999 --config-file mkdocs.insiders.yml
```